codespell_lib/data/dictionary.txt

```diff
@@ -28206,8 +28303,14 @@ incorect->incorrect
 incorectly->incorrectly
 incoropate->incorporate
 incoropates->incorporates
+incoroporate->incorporate
 incoroporated->incorporated
+incoroporates->incorporates
+incoroporating->incorporating
```

Comment:
Add `incorporation` typos?

Reply:
Done. I also added `incooperating->in cooperating` and `incooperation->in cooperation`